tests/integration_tests/security/test_jail.py

```diff
@@ -462,12 +462,12 @@ def test_node_cgroups(test_microvm_with_initrd, sys_setup_cgroups):
         )
 
 
-def test_cgroups_without_numa(test_microvm_with_initrd, sys_setup_cgroups):
+def test_cgroups_without_numa(test_microvm_with_api, sys_setup_cgroups):
     """
     Test the cgroups are correctly set by the jailer, without numa assignment.
     """
     # pylint: disable=redefined-outer-name
```

Comment:
nit: since "# pylint: disable=redefined-outer-name" is moved to the top should we remove all occurrences in this file?

Reply:
fixed